### PR TITLE
1.1.0 readme

### PR DESCRIPTION
**Pull request by selvasingh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
-[![Build Status](https://travis-ci.org/Azure/azure-sdk-for-java.svg?style=flat-square&label=build)](https://travis-ci.org/Azure/azure-sdk-for-java)
+[![Build Status](https://travis-ci.org/Azure/azure-sd1-for-java.svg?style=flat-square&label=build)](https://travis-ci.org/Azure/azure-sdk-for-java)
```
Comment (jianghaolu) [inline README.md]: typo

File: README.md [hunk 2]
```diff
@@ -413,7 +444,7 @@ If you are using released builds from 1.0.0, add the following to your POM file:
 
 ## Help
 
-If you are migrating your code to 1.0.0, you can use these notes for [preparing your code for 1.0 from 1.0 beta 5](./notes/prepare-for-1.0.0.md).
+If you are migrating your code to 1.1.0, you can use these notes for [preparing your code for 1.0 from 1.1.0](./notes/prepare-for-1.1.0.md).
```
Comment (jianghaolu) [inline README.md]: preparing your code for 1.1.0 from 1.0

File: README.md (hunk 1 above)
Comment (selvasingh) [inline README.md]: done

File: README.md (hunk 2 above)
Comment (selvasingh) [inline README.md]: done

File: README.md [hunk 3]
```diff
@@ -249,6 +264,25 @@ You can find plenty of sample code that illustrates management scenarios (69+ en
...
+<li><a href="https://github.com/Azure-Samples/aad-java-manage-service-principals">Manage service principals using Java</a></li>
+<li><a href="https://github.com/Azure-Samples/aad-java-browse-graph-and-manage-roles">Browse graph (users, groups and members) and managing roles</a></li>
+<li><a href="https://github.com/Azure-Samples/aad-java-manage-passwords">Manage passwords</li>
+</ul></td>
+  </tr>
+<tr>
+    <td>Container Service and Container Registry</td>
+    <td><ul style="list-style-type:circle">
+<li><a href="https://github.com/Azure-Samples/acr-java-manage-azure-container-registry">Manage container registry</a></li>
+<li><a href="https://github.com/Azure-Samples/acs-java-deploy-image-from-acr-to-swarm">Deploy an image from container registry to Kubernetes cluster</a></li>
```
Comment (jianghaolu) [inline README.md]: bad link?

File: README.md [hunk 4]
```diff
@@ -249,6 +264,25 @@ You can find plenty of sample code that illustrates management scenarios (69+ en
...
+<li><a href="https://github.com/Azure-Samples/compute-java-manage-virtual-machine-scale-set-with-unmanaged-disks">Deploy an image from container registry to Swarm cluster</li>
```
Comment (jianghaolu) [inline README.md]: same here

**Pull request merged**